.github/ISSUE_TEMPLATE/false-positive.md

```diff
@@ -0,0 +1,21 @@
+---
+name: False positive
+about: This template is used to describe false positive reports.
+title: "[FALSE POSITIVE]: "
+labels: false positive
+assignees: mlevesquedion
```

Comment:
I'd like to see consistency across the templates.  If you want label `false positive` here, use `false negative` in the other.  Alternatively, both are bugs.

Do you mean for these to autoassign to you?

Reply:
Sounds good. I think more granularity could be useful, so I'll create a false negative tag.

> Do you mean for these to autoassign to you?
Yes. I figured it would be good to have it autoassign to _someone_ so we would know about issues as soon as possible. I put my username as a default. Since we can have more than one autoassignee, would you like to be autoassigned as well? Or some other method?

Reply:
It's something we could iterate on.  We could start more coarse, and if we find ourselves with a large influx of reports, add granularity down the line.  Hopefully, they're not tags that are going to need much use.

Reply:
Cross-referencing the other PRs again, they're also fine as is.  We're not overburdened with tags right now, either.

Reply:
(Don't you love how a single carriage return doesn't break the quote formatting?  I banner-blind'd that block.)

I've set a `Watch` on the repo, so I think I should get a notification anytime an Issue is opened.  My concern is mostly a logistical one.  Since GitHub doesn't have a way to distinguish between `Assigned to` and `Actively worked on by`, there wouldn't be a clear "this is a ticket that could be started on by anyone" state of having no assignee.  So while it's fine to auto-assign to you while it's just us working in this space, since I will just ping you and ask if I should pick up an Issue assigned to you.

\shrug

Reply:
Oh wow sorry about the quote block. Fixed.

I think for now I'll set up a `Watch` too and remove the auto-assign since that will allow us to spot which issues we haven't looked into yet.